rust/src/mqtt/mqtt.rs

```diff
@@ -112,7 +112,7 @@ pub struct MQTTState {
     connected: bool,
     skip_request: usize,
     skip_response: usize,
-    max_msg_len: usize,
+    max_msg_len: u32,
```

Comment:
Why this change ?

Reply:
I didn't see a good reason for keeping `max_msg_len` as a `usize` (which seems more for pointers than scalars).